src/components/BookingForm.vue

```diff
@@ -31,26 +34,29 @@
 
 <script>
 import BadContainedButton from "@/components/BadContainedButton";
+import BadTextInput from "@/components/BadTextInput";
+
 export default {
   name: "BookingForm",
   data() {
     return {
       officeId: "",
-      bookingdate: "",
-      emailaddress: ""
+      bookingDate: "",
+      emailAddress: ""
     };
   },
   methods: {
     submitBooking() {
       this.$store.dispatch("book", {
-        officeId: this.officeId,
-        bookingdate: this.bookingdate,
-        emailaddress: this.emailaddress
+        office: { id: this.officeId },
```

Comment:
Updated to match API specs
https://broadsign.slack.com/archives/C01AU5P352L/p1607719055005400